<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widget
{
border-image: url(:/resources/background/1.png);
}
</string>
     </property>
     <widget class="QLabel" name="label1_1">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>251</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);



border-radius: 15px;
background-color: rgb(71,100,104);
color: rgb(240, 240, 240);
</string>
      </property>
      <property name="text">
       <string>  1号：</string>
      </property>
     </widget>
     <widget class="ImageSwitch" name="imageSwitch1_1" native="true">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>70</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label1_2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>60</y>
        <width>251</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);



border-radius: 15px;
background-color: rgb(71,100,104);
color: rgb(240, 240, 240);
</string>
      </property>
      <property name="text">
       <string>  2号：</string>
      </property>
     </widget>
     <widget class="ImageSwitch" name="imageSwitch1_2" native="true">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>70</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label1_4">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>140</y>
        <width>251</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);



border-radius: 15px;
background-color: rgb(71,100,104);
color: rgb(240, 240, 240);
</string>
      </property>
      <property name="text">
       <string>  4号：</string>
      </property>
     </widget>
     <widget class="ImageSwitch" name="imageSwitch1_3" native="true">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>150</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label1_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>140</y>
        <width>251</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);



border-radius: 15px;
background-color: rgb(71,100,104);
color: rgb(240, 240, 240);
</string>
      </property>
      <property name="text">
       <string>  3号：</string>
      </property>
     </widget>
     <widget class="ImageSwitch" name="imageSwitch1_4" native="true">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>150</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label1_5">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>220</y>
        <width>251</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);



border-radius: 15px;
background-color: rgb(71,100,104);
color: rgb(240, 240, 240);
</string>
      </property>
      <property name="text">
       <string>  5号：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label2_1">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>70</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/resources/background/ico_down.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="ImageSwitch" name="imageSwitch1_5" native="true">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>230</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label1_6">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>220</y>
        <width>251</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border-radius: 15px;
background-color: rgb(71,100,104);
color: rgb(240, 240, 240);
</string>
      </property>
      <property name="text">
       <string>  6号：</string>
      </property>
     </widget>
     <widget class="ImageSwitch" name="imageSwitch1_6" native="true">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>230</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label2_4">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>150</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/resources/background/ico_down.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label2_5">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>230</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/resources/background/ico_down.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label2_2">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>70</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/resources/background/ico_down.png);
border-image: url(:/resources/background/ico_down.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label2_3">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>150</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/resources/background/ico_down.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label2_6">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>230</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/resources/background/ico_down.png);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ImageSwitch</class>
   <extends>QWidget</extends>
   <header location="global">imageswitch.h</header>
   <slots>
    <signal>clicked()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
